eZ/Publish/Core/FieldType/Time/Type.php

```diff
@@ -154,6 +154,25 @@ public function fromHash($hash)
     }
 
     /**
+     * Returns if the given $value is considered empty by the field type.
+     *
+     *
+     * @param \eZ\Publish\Core\FieldType\Value $value
+     *
+     * @return bool
+     */
+    public function isEmptyValue(SPIValue $value)
+    {
+        if (empty($value)) {
+            return true;
+        } elseif (!isset($value->time)) {
```

Comment:
this is a bit unclear, `$value->time === null` would be better, which afaik should be fine as *"The given $value can be safely assumed to have already been processed by {@link acceptValue()}."*.